README.md

```diff
@@ -1,21 +1,29 @@
 # Rest API Salary Calculator
 
-The rest api has been created to calculate net amount in pln and display all the taxation applied to the gross amount.
-The application can perform 2 GET  and 1 POST html method, the first two are accessible in the following links:
-* The GET method show the departments and the job title list avaialable, to check them go to the following endpoint:
-  * http://localhost:8080/calculate/getJobDepartment
-  * http://localhost:8080/calculate/getJobTitles/{departmentName}
- 
+The rest api has been created to calculate net amount in pln and display all the taxation applied to
+the gross amount.
+The application can perform 2 GET and 1 POST html method, the first two are accessible in the
+following links:
 
-* The POST html method is used to calculate the salary and accept optional query parameters, go to the following endpoint to calculate your net or participate to the statistics:
+* The GET method show the departments and the job title list avaialable, to check them go to the
+  following endpoint:
+    * http://localhost:8080/calculate/getJobDepartment
```

Comment:
A good practice is to have endpoints like nouns and verbs that should be HTTP methods. I would rethink verb calculate in endpoint and also remove get from JobDepartment. The second thing I would avoid lowerCamelCase in endpoints.

Reply:
Done